README.md

```diff
@@ -20,7 +20,39 @@ var morgan = require('morgan')
 
 Create a new morgan logger middleware function using the given `format` and `options`.
 The `format` argument may be a string of a predefined name (see below for the names),
-a string of a format string, or a function that will produce a log entry.
+a string of a format string, or a format function that will produce a log entry.
```

Comment:
Sure. Changed this in [1c9416c](https://github.com/expressjs/morgan/pull/125/commits/1c9416c7692b359b2f4bcfe7b9b357fe14dc9306). Not sure why this Github comment wasn't hidden automatically because the line was changed.